libsolidity/codegen/MultiUseYulFunctionCollector.cpp

```diff
@@ -43,6 +46,7 @@ string MultiUseYulFunctionCollector::createFunction(string const& _name, functio
 {
 	if (!m_requestedFunctions.count(_name))
 	{
+		m_requestedFunctions[_name] = "STUB";
 		string fun = _creator();
 		solAssert(!fun.empty(), "");
 		solAssert(fun.find("function " + _name) != string::npos, "Function not properly named.");
```

Comment:
```suggestion
		solAssert(fun.find("function " + _name + "(") != string::npos, "Function not properly named.");
```
(why not sneak in something else?)